src/components/Services/Services.tsx

```diff
@@ -16,10 +24,12 @@ interface IVerticalTabs {
 }
 
 const Services: React.FC = () => {
+  const intl = useIntlMessages()
+
   const subitemsDesign: Array<IVerticalTabs> = [
     {
-      key: '1',
-      label: 'Development of graphic pieces',
+      key: 'asd',
```

Comment:
why asd ?

Reply:
it doesn't matter, since it is just a testing key